docs/book/src/architecture/agents.md

```diff
@@ -26,6 +27,23 @@ To allow agents to perform operations in the namespace, they need an identity (S
 [primazactl](https://github.com/primaza/primazactl) is an in-development companion tool to help administrators configuring clusters and namespaces.
 
 
+# Configure agent
+
+Application agent and Service agent deployent are configured in Primaza's Control Plane at runtime.
```

Comment:
Application agent and Service agent deployment are configured in Primaza's Control Plane at runtime.

Reply:
The deployment spelling needs to be corrected